### ISSUE TITLE
Critical issue, cant start server

### ISSUE THREAD

**Issue by PhanaticD** [drtshock/Factions#951]
```
[04:42:09] [Server thread/INFO]: [Factions] Enabling Factions v1.6.9.5-U0.1.21
[04:42:09] [Server thread/INFO]: [Factions v1.6.9.5-U0.1.21] === ENABLE START ===
[04:42:09] [Server thread/ERROR]: Error occurred while enabling Factions v1.6.9.5-U0.1.21 (Is it up to date?)
java.lang.NullPointerException
    at com.massivecraft.factions.zcore.persist.json.JSONFPlayers.loadCore(JSONFPlayers.java:92) ~[?:?]
    at com.massivecraft.factions.zcore.persist.json.JSONFPlayers.load(JSONFPlayers.java:73) ~[?:?]
    at com.massivecraft.factions.P.onEnable(P.java:78) ~[?:?]
    at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:271) ~[patched_1.11.2.jar:git-Paper-1102]
    at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:316) [patched_1.11.2.jar:git-Paper-1102]
    at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:407) [patched_1.11.2.jar:git-Paper-1102]
    at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugin(CraftServer.java:378) [patched_1.11.2.jar:git-Paper-1102]
    at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugins(CraftServer.java:328) [patched_1.11.2.jar:git-Paper-1102]
    at net.minecraft.server.v1_11_R1.MinecraftServer.t(MinecraftServer.java:438) [patched_1.11.2.jar:git-Paper-1102]
    at net.minecraft.server.v1_11_R1.MinecraftServer.l(MinecraftServer.java:399) [patched_1.11.2.jar:git-Paper-1102]
    at net.minecraft.server.v1_11_R1.MinecraftServer.a(MinecraftServer.java:337) [patched_1.11.2.jar:git-Paper-1102]
    at net.minecraft.server.v1_11_R1.DedicatedServer.init(DedicatedServer.java:276) [patched_1.11.2.jar:git-Paper-1102]
    at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:612) [patched_1.11.2.jar:git-Paper-1102]
    at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]
[04:42:09] [Server thread/INFO]: [Factions] Disabling Factions v1.6.9.5-U0.1.21
[04:42:09] [Server thread/INFO]: [Factions v1.6.9.5-U0.1.21] Disabled
```

**Comment by PhanaticD** [comment]
fixed by this.. https://github.com/drtshock/Factions/pull/761

**Comment by MarkehMe** [comment]
This has been fixed in the 1.6 alternative LegacyFactions: https://www.spigotmc.org/resources/legacyfactions.40122/
